### ISSUE TITLE
pip install issue

### ISSUE THREAD

**Issue by yangcal** [NVIDIA/cuda-quantum#1393]
### Required prerequisites

- [X] Consult the [security policy](https://github.com/NVIDIA/cuda-quantum/security/policy). If reporting a security vulnerability, do not report the bug using this form. Use the process described in the policy to report the issue.
- [X] Make sure you've read the [documentation](https://nvidia.github.io/cuda-quantum/latest). Your issue may be addressed there.
- [X] Search the [issue tracker](https://github.com/NVIDIA/cuda-quantum/issues) to verify that this hasn't already been reported. +1 or comment there if it has.
- [X] If possible, make a PR with a failing test to give us a starting point to work on!

### Describe the bug

Hello,

I'm running into issues when I try to install with pip using a fresh environment with `python3.10`.

I expected `pip install cuda-quantum` to resolve all dependencies but I'm hitting the error below after installation

```
>>> import cudaq
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/yangg/software/miniconda3/envs/cudaq/lib/python3.10/site-packages/cudaq/__init__.py", line 28, in <module>
    from ._pycudaq import *
ImportError: Invalid simulator requested: custatevec_fp32
>>> exit()
```

I had to specifically install a few extra dependencies via `pip install nvidia-cublas-cu11 nvidia-cuda-runtime-cu11` to be able to resolve this.

### Steps to reproduce the bug

`pip install cuda-quantum` followed by `python -c "import cudaq"`

### Expected behavior

import should execute just fine

### Is this a regression? If it is, put the last known working version (or commit) here.

Not a regression

### Environment

- **CUDA Quantum version**:  0.06
- **Python version**: 3.10
- **C++ compiler**: Unknown
- **Operating system**: Linux

**Comment by bmhowe23** [comment]
Hi @yangcal - thanks for filing the issue. I think the problem is that those pip packages aren't available for ARM, so our [PyPi instructions](https://pypi.org/project/cuda-quantum/) discuss having to install those dependencies ahead of time. One thing added in 0.7.0 (which will be released soon) is PR #1152, which will fall back on the CPU simulators if not all the dependencies are installed. That probably doesn't give you exactly what you want, but I figured I'd mention it anyway.

We'll leave this issue open to see if we can come up with a better solution.

**Comment by bettinaheim** [comment]
Closing this, as we cannot/won't include CUDA dependencies in python wheels due to their size, and python wheels are not available for all necessary cuda dependencies. A future alternative were we shouldn't have a similar issue is distribution of and installation via conda (not yet available).
